src/devrules/cli_commands/group.py

```
@@ -0,0 +1,640 @@
+            if not should_run:
+                typer.secho("Skipping...", fg=typer.colors.YELLOW)
+                return False
+
+            cmd_str = " ".join(command)
+            typer.secho(f"Running: {cmd_str}", fg=typer.colors.BLUE)
+
+            if dry_run:
+                return True
+
+            try:
+                subprocess.run(command, check=check)
```

Ruff (0.14.8)

`subprocess` call: check for execution of untrusted input

(S603)